package.json

```diff
@@ -25,12 +27,16 @@
     "@types/node": "^12.20.17",
     "@types/react": "^17.0.15",
     "@types/react-dom": "^17.0.9",
+    "dompurify": "^2.3.6",
+    "history": "^5.3.0",
+    "marked": "^4.0.12",
```

Comment:
You are correct, I didn't need to install this library yet but it will be needed when importing navigator into Europa. I can uninstall but we will probably need it pretty soon. [Here's](https://github.com/kbase/navigator/blob/main/src/client/components/dashboard/NarrativeList/Preview.tsx#L189) where its used